HackAtUCI/irvinehacks-site
# [Admin] 2 Hacker reviewers

- Affected `/applicants`, `/hackerApplicants`, `/release`, `/releaseHackerDecisions`, `/review`, `/hackerReview`, `/setThresholds`, route
- Still needs way to show an applicant's decision on frontend `HackerApplicants.tsx`
- Still needs way to filter accepted, waitlisted, rejected and other filter

## Testing
- Create a sample app in the db
- Navigate to that app, score it in the top right
  - Ensure the score updates in the reviews log on that page, updates in the db, updates the number of reviewers in the applicants overview, and status is still set to pending_review
- Review that app again with a different account
  - Ensure avg_score updates in db, status is set to reviewed
- Manually run `/releaseHackerDecisions` to ensure db status updates
